modules/imaging_browser/php/NDB_Form_imaging_browser.class.inc

```diff
@@ -51,7 +51,7 @@ class NDB_Form_Imaging_Browser extends NDB_Form
         // allow only to view own site data
         $timePoint =& TimePoint::singleton($_REQUEST['sessionID']);
 
-        return ($user->hasPermission('imaging_browser_view_allsites') || ((in_array($timePoint->getData('CenterID'), $user->getData('CenterIDs'))) && $user->hasPermission('imaging_browser_view_site')));
+        return ($user->hasPermission('imaging_browser_view_allsites') || ((in_array($timePoint->getData('CenterID'), $user->getData('CenterIDs'))) && $user->hasPermission('imaging_browser_view_site')) || $user->hasPermission('imaging_browser_phantom_data'));
```

Comment:
Not sure if this is intended behaviour, but this will return true for a user who _doesn't_ have `imaging_browser_view_allsites`, but _does_ have `imaging_browser_phantom_data`, regardless of whether or not they have the Timepoint's CenterID in their user object.

In other words, if `imaging_browser_phantom_data` is true, this expression will always return true. Again, I'm not sure if this is intended. If not, I think the line should have an extra set of parenthesis:

```
return ($user->hasPermission('imaging_browser_view_allsites') || ((in_array($timePoint->getData('CenterID'), $user->getData('CenterIDs'))) && ($user->hasPermission('imaging_browser_view_site')) || $user->hasPermission('imaging_browser_phantom_data')));
```